notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>General comment: Each time you say "You will use these helper functions" or something similar, please remove "You will"</p><p>(https://developers.google.com/style/tense)</p><p> </p><p>An example here:</p><p><s>You will</s> "Use these helper functions to load the images from the <code>TFRecord</code>s into a <code>tf.data.Dataset</code>."</p><p> </p><p>General comment: Put "TFRecord" in code formatting each time you mention it throughout the notebook. The plural is <code>TFRecord</code> s.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>